tokio-console/src/config.rs

```diff
@@ -95,6 +96,17 @@ pub enum OptionalCmd {
     ///     $ tokio-console gen-config > console.toml
     ///
     GenConfig,
+
+    /// Generate shell completions
+    ///
+    /// It should be saved in expected directory, depending on the shell used
```

Comment:
maybe something like this:
```suggestion
    /// The completion script should be saved in the shell's completion directory.
    /// This depends on which shell is in use.
    ///
    /// For example, using zsh:
```

Reply:
fixed: https://github.com/tokio-rs/console/pull/336/commits/60fbce5744bf9776157cfbcfce9d714fefb7f460